Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:6-slim
+
+RUN apt-get update && apt-get install libpng12-0
+
+WORKDIR /reactapp
+
+RUN mkdir -p /reactapp
```

Comment:
This line is unnecessary. 
WORKDIR creates the path if it doesn't exist:  https://docs.docker.com/engine/reference/builder/#workdir